docs/source/users/problem_definition_files/spinw.rst

```diff
@@ -0,0 +1,120 @@
+.. _spinw_format:
+
+*****************
+SpinW File Format
+*****************
+
+The SpinW file format is based on :ref:`native`, this page is intended to
+demonstrate where the format differs.
+
+As in the native format, an input file must start with a comment indicating
+that it is a FitBenchmarking problem followed by a number of key value pairs.
+Available keys can be seen in :ref:`native` and below:
+
+software, name, description
+  As described in the native format.
+
+input_file
+  For SpinW we require an SQW file containing the data from Horace.
```

Comment:
So the [datafile](https://github.com/fitbenchmarking/fitbenchmarking/pull/1049/files#diff-07896593f94f5ca1aa84a414b0eac73fc4eac357d6d8d70de5dcf2096568b14fR5) used here (size 180MB) is actually on the _small_ size for typical Horace datasets (~10-100GB) but obviously still too big to be included in the repo. Even just cutting down to the [fraction used](https://github.com/fitbenchmarking/fitbenchmarking/pull/1049/files#diff-07896593f94f5ca1aa84a414b0eac73fc4eac357d6d8d70de5dcf2096568b14fR7) in this fitting it is still 57MB.

Now, Horace does have a function called `fake_sqw` ([code here](https://github.com/pace-neutrons/Horace/blob/527d074f962d81c0ce345e22dae1d14ccf446bc6/horace_core/algorithms/fake_sqw.m), and [documentation here](https://pace-neutrons.github.io/Horace/v3.6.2/manual/Planning_a_Horace_scan.html?highlight=fake_sqw#more-detailed-planning-of-scans)) which allows you to generate (blank) data files. You can then use the `sqw_eval` function ([docs](https://pace-neutrons.github.io/Horace/v3.6.2/manual/Simulation.html?highlight=sqw_eval#sqw-eval)) to populate the data file with model calculations and add some Gaussian noise.

You do need some extra information for this: 

* The detector positions for the instrument you want to simulate
* Details of the experiment you want to simulate (incident neutron energy, crystal orientations, crystal lattice parameters)
[gen_spinw_fake.m.txt](https://github.com/fitbenchmarking/fitbenchmarking/files/9185271/gen_spinw_fake.m.txt)

* A SpinW model to generate the simulated data

It's probably best to put this into a separate input file. So, e.g. if users specific `<some_file>.sqw` then the code just does what it does now and reads from this file. If they users specify `<some_file>.sim` (or some such extension - maybe `.sim.yaml` or `.sim.json` or something) then this will be a text file with the parameters needed...

I've created a Matlab script ([gen_spinw_fake.m.txt](https://github.com/fitbenchmarking/fitbenchmarking/files/9185272/gen_spinw_fake.m.txt) - renamed to `txt` to satisfy github...) which will generate the input file - unfortunately, it does take quite a long time to run (around 6h on the small IDAaaS instance), so it might not be the solution to your problem...